server/.env.example

```
@@ -1,59 +1,15 @@
-# Firestore Emulator を使用する場合（ローカル開発用）
-# Docker環境では compose.yml で自動設定されます
-# FIRESTORE_EMULATOR_HOST=localhost:8080
-
-# Firestore プロジェクトID（通常は GCP_PROJECT_ID と同じ）
-# FIRESTORE_PROJECT_ID=dev-project
-
-# -------------------------------------------
-# LLMモデル設定
-# -------------------------------------------
-# ファインチューニング済みモデルのエンドポイントID（必須）
+LLM_PROVIDER= # gemini or grok
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)